sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -514,7 +537,6 @@ export class EventProcessor {
       this._abortController.abort();
     }
 
```

Comment:
A goofy edge case but removing this means that if I do this:

const ep = new EP();
ep.stop()

I'll still be able to use the client since no processors were running when I stop'd (and thus nobody would toggle the _isRunning flag for me).

Reply:
In this case wouldn't `_isRunning` be false or undefined, since `subscribe` was never called? I thought we supported calling `start` after `stop`.

Reply:
So I took a look at the `start()` method:
https://github.com/Azure/azure-sdk-for-js/blob/f4cd3a5d365cc67a7f9d8579e66c55559dfcc55f/sdk/eventhub/event-hubs/src/eventProcessor.ts#L474-L480

I think this change actually fixes an edge case. Before this change, it would be possible to call `start()` immediately after calling `stop()`. Since `this._isRunning` was set to false right away in the `stop` method, the EventProcessor could start its loop while it was still being torn down. Now `this._isRunning` is set to false only after the partition pumps have been removed and the running loop exits.